forth_src/c64/kernal.fs

```diff
@@ -0,0 +1,4 @@
+\ Definitions for kernal vars and routines
+
+\ TBLX - cursor line number
+$d6 constant k-tblx
```

Comment:
is "k" for "kernal"? i think "tblx" would be enough.

It's a little disturbing to have a file for a single constant, since compile times increase with the number of files. Could it be solved in some other way?